osm2lanes/slides.md

```diff
@@ -49,27 +49,27 @@ format:
 :::
 ::::
 
-## Edit roads
+## Edit Roads
 
 ![](https://a-b-street.github.io/docs/project/history/retrospective/edit_roads.gif)
 
-## Simulate traffic
+## Simulate Traffic
 
 ![](https://a-b-street.github.io/docs/project/history/retrospective/traffic_sim.gif)
 
-## Plan bike networks
+## Plan Bike Networks
 
 [bike.abstreet.org](http://bike.abstreet.org)
 
 ![](https://a-b-street.github.io/docs/software/ungap_the_map/demo.gif)
 
-## Low-traffic neighborhoods
+## Low-traffic Neighborhoods
```

Comment:
I low-key enjoy the quiet tension in the room every time I use American spelling at work ;)

Reply:
Oh no, I didn't even notice. Would have corrected it if I had :P